charts/gha-runner-scale-set-controller/templates/manager_role.yaml

```diff
@@ -146,6 +146,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
```

Comment:
do you need to have both `list` and `watch`? maybe `get` is enough?

Reply:
if we are not owning configmap or watching on it, as far as I know, get should be enough

Reply:
Client needs both because it caches by default. should we maybe also revisit as part of the work on permissions?